lib/stormpath-sdk.rb

```diff
@@ -69,6 +70,7 @@ module Authentication
     autoload :UsernamePasswordRequest, "stormpath-sdk/auth/username_password_request"
     autoload :BasicLoginAttempt, "stormpath-sdk/auth/basic_login_attempt"
     autoload :AuthenticationResult, "stormpath-sdk/auth/authentication_result"
+    autoload :JwtAuthenticationResult, "stormpath-sdk/auth/jwt_authentication_result"
```

Comment:
I would move this to the Oauth module and probably rename it to OauthAuthenticationResult...or AccessTokenAuthenticationResult. This is because Jwt is too specific to the way Stormpath is handling the token and the user doesn't have to know about it.